Release 1.10.0

e-rk opened the pull request.

hubertmis commented on src/raal/softdevice/nrf_raal_softdevice.c:
```diff
@@ -191,23 +192,23 @@ static void timeslot_request(void)
-    NRF_TIMER0->CC[TIMER_CC_EXTEND] = 0;
-    NRF_TIMER0->CC[TIMER_CC_MARGIN] = m_timeslot_length - m_config.timeslot_safe_margin;
-    NRF_TIMER0->TASKS_START         = 1;
+    nrf_timer_task_trigger(NRF_TIMER0, NRF_TIMER_TASK_STOP);
```
Perhaps, we could provide `#define RAAL_TIMER NRF_TIMER0` in case we would use other timer instance in future?

e-rk committed:
Release v1.10.0

Source commits:
nrf-802.15.4-driver: 7c64e834eee498100f3b80b9e064381be848c470
nrf-802.15.4-sl:     4fc6d7ec134f32137a11b0d7e3826ae449c66875

hubertmis commented on nrf_radio_802154:
```diff
@@ -0,0 +1,51 @@
+set(NRF_802154_DRIVER_INCLUDE_DIRS ${NRF_802154_DRIVER_ROOT}/src)
```
Is it a cmake file? Should it have `.cmake` suffix or something like this?

e-rk replied:
That file was added by mistake.

hubertmis reviewed (commented):
Looks very good! Found one file without suffix. Not sure if it is intentional.

jciupis commented on nrf_radio_802154:
```diff
@@ -0,0 +1,51 @@
+set(NRF_802154_DRIVER_INCLUDE_DIRS ${NRF_802154_DRIVER_ROOT}/src)
```
This file seems to be pushed by mistake

e-rk replied:
You are right. Removed.

hubertmis commented:
BTW, diff shows 0 lines changed in SL binary libraries. Please, double- check if new version is committed. I'm afraid .a files are filtered out by .gitignore.

hubertmis commented:
I think it would be better to squash commits to introduce compatible driver and SL as a single change. Also, please add SHA of source repositories to the commit message.

hubertmis reviewed (changes requested):
Please add SHAs of commits in the source repository in the commit message and we'll be good to go!

e-rk commented:
The libraries seem to be in order. Here are checksums from libraries in the repo and libraries downloaded from the build artifact.
```
% find . -name "*.a" -exec sha256sum {} \; | sort
27948ad2e27d80d86dad1573a64755e9371d01be43944306c1f6c95aa55dc70b  ./nRF52833/soft-float/libnrf_802154_sl.a
48a334c7b2a168abc93255e654803f2c4312ebe16139a451ef107b0bb8349d64  ./nRF52811/soft-float/libnrf_802154_sl.a
4b846716950fdc4ab1ae77c3d4d2d4aee2f67a215f5128816f72712175dfe299  ./nRF52833/softfp-float/libnrf_802154_sl.a
7c4e426afe5af6577dbddce1393e27023d8f487cce478544c1340f4a7a484d15  ./nRF52833/hard-float/libnrf_802154_sl.a
988b469684e21b62d9fcef920138d734a8c106a15c8816a075d1fc7b2837acb7  ./nRF52840/softfp-float/libnrf_802154_sl.a
c2e8690c88c8213477cc9390cf5d384fa54a3f26de193873408f613bc935ccb0  ./nRF52840/soft-float/libnrf_802154_sl.a
cdf6420590163d722b04827e5c96ec4a10df083b8b7ea1b411cc4fbc44551985  ./nRF52840/hard-float/libnrf_802154_sl.a
ee3d4a6645d347c33dd44309c9f98ebbdb37f4aa430bf1ff9efccfb26be61633  ./nRF5340_cpunet/soft-float/libnrf_802154_sl.a
f58dd431c241dca2d404a9937f4134d9efea1cbae54e9ebf7daa0e059cb0a757  ./nRF52820/soft-float/libnrf_802154_sl.a
```

```
% find . -name "*.a" -exec sha256sum {} \; | sort 
27948ad2e27d80d86dad1573a64755e9371d01be43944306c1f6c95aa55dc70b  ./nrf52833_xxAA/soft/libnrf_802154_sl.a
48a334c7b2a168abc93255e654803f2c4312ebe16139a451ef107b0bb8349d64  ./nrf52811_xxAA/soft/libnrf_802154_sl.a
4b846716950fdc4ab1ae77c3d4d2d4aee2f67a215f5128816f72712175dfe299  ./nrf52833_xxAA/softfp/libnrf_802154_sl.a
7c4e426afe5af6577dbddce1393e27023d8f487cce478544c1340f4a7a484d15  ./nrf52833_xxAA/hard/libnrf_802154_sl.a
988b469684e21b62d9fcef920138d734a8c106a15c8816a075d1fc7b2837acb7  ./nrf52840_xxAA/softfp/libnrf_802154_sl.a
c2e8690c88c8213477cc9390cf5d384fa54a3f26de193873408f613bc935ccb0  ./nrf52840_xxAA/soft/libnrf_802154_sl.a
cdf6420590163d722b04827e5c96ec4a10df083b8b7ea1b411cc4fbc44551985  ./nrf52840_xxAA/hard/libnrf_802154_sl.a
ee3d4a6645d347c33dd44309c9f98ebbdb37f4aa430bf1ff9efccfb26be61633  ./nrf5340_cpunet_xxAA/soft/libnrf_802154_sl.a
f58dd431c241dca2d404a9937f4134d9efea1cbae54e9ebf7daa0e059cb0a757  ./nrf52820_xxAA/soft/libnrf_802154_sl.a
```

hubertmis commented:
@e-rk I meant hashes of commits in internal repositories. The commits you copied files from and built libraries from. Please have a look at previous commit messages in this repository for reference.

e-rk commented:
Yes, I knew that. I was in the middle of doing that. I copied the comparison just to remove any doubt that the libraries might have been ignored by .gitignore.

hubertmis reviewed (approved):
Perfect now!
